src/routes/index.ts

```diff
@@ -39,15 +39,21 @@ const isValidEmail = (email: string) => {
   return emailRegex.test(email);
 };
 
+const isValidPhone = (phone: string) => {
+  // eslint-disable-next-line no-control-regex
+  const phoneRegex = /^\+[0-9]{10,15}$/;
+  return phoneRegex.test(phone);
+};
+
 const EmailPromptRequest = z.object({
   email: z.string(),
   recaptcha: z.string(),
 });
 
 const handleEmailPromptSubmission: RequestHandler<{}, z.infer<typeof EmailPromptRequest>, { email: string, encrypted: string, error?: string }> = async (req, res, _next) => {
   const { email, recaptcha } = EmailPromptRequest.parse(req.body);
-  if (!isValidEmail(email)) {
-    res.render('index', { email, error : i18n.__('Please enter a valid email address') });
+  if (!isValidEmail(email) && !isValidPhone(email)) {
+    res.render('index', { email, error : i18n.__('Please enter a valid email address or phone number') });
```

Comment:
Should we give an example of the valid formatting? It's really frustrating to type `1-555-555-5555` and have it reject because it didn't have dashes, and you didn't know that.